src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -79,56 +79,39 @@ public function __construct($decorated = false, array $styles = array())
     }
 
     /**
-     * Sets the decorated flag.
-     *
-     * @param bool $decorated Whether to decorate the messages or not
+     * {@inheritdoc}
      */
     public function setDecorated($decorated)
     {
         $this->decorated = (bool) $decorated;
     }
 
     /**
-     * Gets the decorated flag.
-     *
-     * @return bool true if the output will decorate messages, false otherwise
+     * {@inheritdoc}
      */
     public function isDecorated()
     {
         return $this->decorated;
     }
 
     /**
-     * Sets a new style.
-     *
-     * @param string                        $name  The style name
-     * @param OutputFormatterStyleInterface $style The style instance
+     * {@inheritdoc}
      */
     public function setStyle($name, OutputFormatterStyleInterface $style)
     {
         $this->styles[strtolower($name)] = $style;
     }
 
     /**
-     * Checks if output formatter has style with specified name.
-     *
-     * @param string $name
-     *
-     * @return bool
+     * {@inheritdoc}
      */
     public function hasStyle($name)
     {
         return isset($this->styles[strtolower($name)]);
     }
 
     /**
-     * Gets style options from style with specified name.
-     *
-     * @param string $name
-     *
-     * @return OutputFormatterStyleInterface
-     *
-     * @throws \InvalidArgumentException When style isn't defined
```

Comment:
I would say it should be. unrelated but this should uses the console domain exception instead